src/test/groovy/com/datadog/ddwaf/LimitsTests.groovy

```diff
@@ -181,8 +169,9 @@ class LimitsTests implements WafTrait {
             ]
           }''')
 
-        ctx = Waf.createHandle('test', atom)
-
+        builder.addOrUpdateConfig('atom', atom)
```

Comment:
Re-enabled this test at https://github.com/DataDog/libddwaf-java/pull/171
Once merged we can rebase here.